index.html

```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <main>
-        <h1>Welcome to Pontify</h1>  
+        <h1>Welcome to Pontify ! Hope you enjoy your stay with us. </h1>  
```

Comment:
Could you remove 'enjoy your stay' string, fix the name Pontify to Portionfy and make 'Welcome to Portinfy' red ?

Reply:
Sure, no problem.

Reply:
Sure @marquitos-santos , I can do it.